node/tests/ServerModules.test.ts

```diff
@@ -2677,7 +2749,65 @@ describe("Server Module Tests", () => {
                         condition: "used",
                         bicycles: isResp3 ? 4 : "4",
                     },
+                ];
+                expect(
+                    aggreg
+                        .map(convertGlideRecordToRecord)
+                        // elements (records in array) could be reordered
+                        .sort((a, b) =>
+                            a["condition"]! > b["condition"]! ? 1 : -1,
+                        ),
+                ).toEqual(expectedAggreg);
+
+                const aggregProfile: [
+                    FtAggregateReturnType,
+                    Record<string, number>,
+                ] = await GlideFt.profileAggregate(
+                    client,
+                    indexBicycles,
+                    "*",
+                    options,
+                );
+                expect(Object.keys(aggregProfile[1])).toEqual([
+                    "parse.time",
+                    "all.count",
```

Comment:
Content of profiling data is subject to change, I'd ignore it. You can do non-null, non-empty checks if you want to verify it.